src/modes_hosgrid.cpp

```diff
@@ -98,12 +96,12 @@ void modes_hosgrid::populate_hos_eta_nondim(
     const int n0, const int n1, fftw_plan p, fftw_complex *eta_modes,
     amrex::Gpu::DeviceVector<amrex::Real> &HOS_eta) {
   // Local array for output data
-  double out[n0 * n1];
+  amrex::Vector<double> out(n0 * n1, 0.0);
```

Comment:
I'm seeing `double` all around here. Are they required?

Reply:
as opposed to "amrex::Real"? Yeah idk what FFT expects

Reply:
didn't really want to deal with that I guess?

Reply:
oh yeah FFTW, let's just fix it for now

Reply:
I mean not convert to amrex::Real